crates/core/tedge_mapper/src/lib.rs

```diff
@@ -122,15 +118,11 @@ pub async fn run(mapper_opt: MapperOpt) -> anyhow::Result<()> {
         tedge_config::TEdgeConfigLocation::from_custom_root(&mapper_opt.config_dir);
     let config = tedge_config::TEdgeConfig::try_new(tedge_config_location.clone())?;
 
-    let log_level = if mapper_opt.debug {
-        tracing::Level::DEBUG
-    } else {
-        get_log_level(
-            "tedge-mapper",
-            &tedge_config_location.tedge_config_root_path,
-        )?
-    };
-    set_log_level(log_level);
+    log_init(
+        "tedge-mapper",
+        &mapper_opt.log_args,
+        &tedge_config_location.tedge_config_root_path,
```

Comment:
Previously erroneously used `&tedge_config_location.tedge_config_file_path` instead of `root_path` here, squashed and rebased a fix.